### DISCUSSION TITLE
Upgrading to from 0.1.0 to 0.3.0 stops CSV panels working

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#20] [Q&A] [answered]
Hello,

I recently upgraded to version 0.3.0 from 0.1.0 and all my CSV panels stopped working which we rely on, so I rolled back via a snapshot.  I didn't see any errors to help debug.

I'd like to try it again, but is there an easier way to downgrade from 0.3.0 to 0.1.0 rather that use a VM snapshot?

Thanks

**Comment by marcusolsson** [comment, marked as answer]
You should be able to [install a specific version of a plugin](https://grafana.com/docs/grafana/latest/administration/cli/#install-a-specific-version-of-a-plugin) with the following command:

```
grafana-cli plugins install <plugin-id> <version>
```
